draft-ietf-quic-ack-frequency.md

```diff
@@ -411,18 +411,30 @@ controllers.
 ## Congestion Control
 
 A sender needs to be responsive to notifications of congestion, such as
-a packet loss or an ECN CE marking. Also, window-based congestion controllers
-that strictly adhere to packet conservation, such as the one defined in
-{{QUIC-RECOVERY}}, rely on receipt of acknowledgments to send additional data into
-the network, and will suffer degraded performance if acknowledgments are delayed
-excessively.
-
-To enable a sender to respond to potential network congestion, a sender SHOULD
-cause a receiver to send an acknowledgement at least once per RTT if there are
-unacknowledged ack-eliciting packets in flight. A sender can accomplish this by
-sending an IMMEDIATE_ACK frame once per round-trip time (RTT), or it can set the
-Ack-Eliciting Threshold and Request Max Ack Delay values to be no more than a
-congestion window and an estimated RTT, respectively.
+a packet loss or an ECN CE marking.  To enable a sender to respond to potential
+network congestion, a sender SHOULD cause a receiver to send an acknowledgement
+at least once per RTT if there are unacknowledged ack-eliciting packets in flight.
+A sender can accomplish this by sending an IMMEDIATE_ACK frame once per
+round-trip time (RTT), or it can set the Ack-Eliciting Threshold and
+Request Max Ack Delay values to be no more than a congestion window and an
+estimated RTT, respectively.
+
+Note that it's possible the RTT is smaller than the receiver's timer granularity,
+as communicated via the 'min_ack_delay' transport parameter, preventing the
+receiver from sending an acknowledgment every RTT in time.  In these cases,
+'Ignore Order' can be harmful, because it delays fast loss detection.
+
+A congestion controller that is congestion window limited relies upon receiving
+acknowledgements to send additional data into the network.  An increase in
+acknowledgement delay increases the delay in sending data, which can reduce the
+achieved throughput.  Congestion window growth can also depend upon receiving
+acknowledgements, such as in slow start ({{Section 7.3.1 of QUIC-RECOVERY}}),
+so delaying acknowledgements can delay the increase in congestion window.
+
+Window-based congestion controllers, such as the one defined in {{QUIC-RECOVERY}},
+rely on receipt of acknowledgments to send additional data into the network,
+so performance is degraded if acknowledgments are delayed excessively.
```

Comment:
Looking again the two are slightly different  and need to be combined.

Both are describing the case when the sender is window-limited - (1) it increases path delay, and hence lowers throughput (2) it delays growth of cwnd, and where performance can be impacted. (and we avoid anyone thinking fewer ACKs always means lower throughput).

Reply:
I think this is well covered in the previous paragraph.